Story/Voltaire/CoreVoltaire.cs

```diff
@@ -0,0 +1,432 @@
+        Story.KillQuest(3102, "vampirates", "Vampirate");
+
+        //Oh Right, They're Vampires 3103
+        Story.KillQuest(3103, "vampirates", new[] { "Creepy Bat", "Vampirate" });
+        
+        //The Vampirate Captain 3104
+        Story.MapItemQuest(3104, "vampirates", 1959);
+        
+        //How Are We Breathing Under Here!? 3105
+        Story.KillQuest(3105, "vampirates", "Stranglerfish");
+        
+        //Unsink This Ship! 3106
+        Story.KillQuest(3106, "vampirates", new[] { "Murderaconian", "Vampirate", "Vampirate" });
+        
+        //Release the Kraken! Wait, no - Don't. 3107
+        Story.KillQuest(3107, "vampirates", "Bracken Kraken");
+        
+        //I Didn't Know They Could Swarm! 3108
+        Story.KillQuest(3108, "vampirates", "Vampirate");
+    }
+
+    public void TreasureIsland()
+    {
+        if (Core.isCompletedBefore(3111))
+            return;
+
+        //Now Where Are Those Landmarks? 3109
+        Story.MapItemQuest(3109, "treasureisland", new[] { 1986, 1987, 1988, 1989 });
+        
+        //X Marks The Spot 3110
+        Story.MapItemQuest(3110, "treasureisland", 1958, 1);
+        
+        //Captain Von Poach 3111
+        Story.KillQuest(3111, "treasureisland", "Captain Von Poach");
+    }
+
+    public void GustavduGrog()
+    {
+        if (Core.isCompletedBefore(3119))
+            return;
+
+        //Me Knickers Got a Big Hole Across th' Bum 3114
+        Story.KillQuest(3114, "skullpunch", new[] { "Vampirate", "Fishbones" });
+
+        //Where'd Me Ship's Cargo? 3115
+        Story.KillQuest(3115, "skullpunch", new[] { "Fishbones", "Fishbones", "Fishbones", "Fishbones" });
+
+        //A Most Important Package 3116
+        Story.KillQuest(3116, "skullpunch", "Fishbones");
+
+        //Comfort Food 3117
+        Story.KillQuest(3117, "skullpunch", new[] { "Shelleton", "Fishwing" });
+
+        //Full o' Holes 3118
+        Story.KillQuest(3118, "skullpunch", "Shelleton");
+
+        //Keelhaulin' a Kraken! 3119
+        Story.KillQuest(3119, "chaoskraken", "Chaos Kraken");
+    }
+
+    public void Wormhole()
+    {
+        //Something In The Air 5051
+        Story.KillQuest(5051, "wormhole", new[] { "Goth Girl", "Vamp Boy" });
+        
+        //They're Not Red Shirts - They're Deadshirts! 5052
+        Story.MapItemQuest(5052, "wormhole", new[] { 4420, 4421, 4422 });
+        Story.KillQuest(5052, "wormhole", "Deadshirt");
+
+        //Curses! 5053
+        Story.KillQuest(5053, "wormhole", "Cursed Alien");
+
+        //Computers Don't Lie 5054
+        Story.MapItemQuest(5054, "wormhole", new[] { 4423, 4424, 4425, 4426 });
+
+        //Beam Me Up! 5055
+        Story.KillQuest(5055, "wormhole", new[] { "Undead Astronaut", "Undead Astronaut" });
+
+        //Exotic Pets 5056
+        Story.KillQuest(5056, "wormhole", "Xenodog");
+
+        //Not Trobbles 5057
+        Story.KillQuest(5057, "wormhole", "Green Trobbolier|Purple Trobbolier"); // Be patient
```

Comment:
be patient?

Reply:
for some reason the bot slowly hunt this monster idk